stores-api.yaml

```diff
@@ -1,95 +1,626 @@
+    GeoJSON:
+      title: GeoJSON
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+          - Point
+        coordinates:
+          type: array
+          description: Array containing longitudea and latitude
+          items:
+            type: number
+          minItems: 2
+          maxItems: 2
+      example:
+        type: Point
+        coordinates:
+        - 13.408134
+        - 52.489908
+    Image:
+      title: Image
+      type: object
+      properties:
+        dimensions:
+          $ref: '#/components/schemas/ImageDimensions'
+        label:
+          type: string
+        url:
+          type: string
+    ImageDimensions:
+      title: ImageDimensions
+      type: object
+      description: Dimensions of the original image. This can be used by your application for example to determine whether the image is large enough to display a zoom view.
+      properties:
+        h:
+          type: integer
+          format: int32
+        w:
+          type: integer
+          format: int32
+    Price:
+      title: Price
+      type: object
+      properties:
+        currencyCode:
+          type: string
+        value:
+          type: number
+    Product:
+      title: Product
       type: object
       properties:
+        description:
+          $ref: '#/components/schemas/LocalizedString'
         id:
-          $ref: '#/components/schemas/Id'
+          type: string
+        masterVariant:
+          $ref: '#/components/schemas/ProductVariant'
+        name:
+          $ref: '#/components/schemas/LocalizedString'
+        price:
+          $ref: '#/components/schemas/Price'
+        slug:
+          $ref: '#/components/schemas/LocalizedString'
+      example:
+        id: 6a19bf78-dce2-4830-845f-b029ada5472e
+        name:
+          de_DE: 2x kleine Hamamtücher 50x90cm
+        description:
+          de_DE: |-
+            Du bekommst 2x kleine Hamamtücher 50x90cm aus 100% Baumwolle perfekt im Bad oder Küche
+            Voraussichtliche Lieferung: ab sofort
+        slug:
+          en_US: 2-hamam-50-90cm-rot-weiss
+          de_DE: 2x-kleine-hamamtuecher-50x90cm
+        price:
+          value: 25
+          currencyCode: EUR
+        masterVariant:
+          assets: []
+          attributes: []
+          id: 1
+          images:
+          - url: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/2-hamam-50-90cm-rot--q9KcG8UT.jpg
+            dimensions:
+              w: 566
+              h: 566
+            label: 2 Hamamtücher 50 x 90cm
+          key: 2-hamam-50-90cm-rot-weiss
+          sku: 2-hamam-50-90cm-rot-weiss
+    ProductDraft:
+      title: ProductDraft
+      type: object
+      properties:
+        description:
+          $ref: '#/components/schemas/LocalizedString'
         name:
+          $ref: '#/components/schemas/LocalizedString'
+        price:
+          $ref: '#/components/schemas/Price'
+        slug:
```

Comment:
The slug should be generated on backend side. It can`t be provided from the frontend (e.g. registration form.)